css/style.css

```diff
@@ -14,19 +22,86 @@ body{
     align-items: center;
 }
 
+#toggle-input{
+    display: none;
+}
+
+@media only screen and (max-width: 499px){
+    .header__top_items > .header__top_items_link{
+        display: none;
+    }
```

Comment:
as we are trying to implement by BEM, pls get rid of such selectors.
PS. maybe you should move your media to the bottom of the file?